contracts/extensions/age003-emergency-execute.clar

```diff
@@ -55,6 +56,7 @@
 (define-public (set-signals-required (new-requirement uint))
 	(begin
 		(try! (is-dao-or-extension))
+		(asserts! (> new-requirement u0) err-invalid-signals)
```

Comment:
I'm not sure if it makes a difference, both `u0` and `u1` would trigger an execution immediately when `executive-action` is called.

Reply:
True, let's remove.